README.md

```diff
@@ -107,3 +107,18 @@ const dataTable = new simpleDatatables.DataTable("#myTable", {
 	...
 })
 ```
+
+---
+
+## How to contribute?
+
+1. Fork the repository
+2. Create a sub-branch
+3. Clone the sub-branch to your local system
+4. Install [NodeJS](https://nodejs.org/en)https://nodejs.org/en
+5. Open the project in [Visual Studio Code](https://code.visualstudio.com/)
```

Comment:
@shaunroselt Can we change that line so it's not advertisement for a specific product? I myself use the Pulsar editor. But any code editor should work. We can still mention Visual Studio Code as one example of an editor. Something like:

5. Open the project in a code editor (for example [Visual Studio Code](https://code.visualstudio.com/) or [Pulsar Edit](https://pulsar-edit.dev/))